### PR TITLE
EZEE-2021: Fields view in Page View mode for Page builder

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-2021
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by lserwatka** [comment]
@dew326 could you run fixer here. Behat is green.

File: src/bundle/Resources/views/content/content_view/content_fields.html.twig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{% set default_fields_group_template = '@EzPlatformAdminUi/content/content_view/fields.html.twig'%}
+{% set fields_group_template = fields_group_template is defined ? fields_group_template : default_fields_group_template %}
```
Comment (mikadamczyk) [inline src/bundle/Resources/views/content/content_view/content_fields.html.twig]: Twig has the null-coalescing operator. I think we can change this to `{% set fields_group_template = fields_group_template ?? default_fields_group_template %}

**Comment by webhdx** [comment]
@mikadamczyk @ViniTou @Nattfarinn please redo your code reviews. Implementation changed with better decoupling.

**Comment by lserwatka** [comment]
@webhdx fixniesz ten problem z CS tutaj i zmergujemy.

**Pull request merged**